config/json.go

```diff
@@ -11,6 +11,8 @@ import (
 	"os"
 )
 
+var ConfigFilePath string
```

Comment:
__[golint]__
* [golint] exported var ConfigFilePath should have comment or be unexported

<!-- Comment by [SideCI](https://sideci.com) -->